lib/ts/ingredients/emaildelivery/services/smtp.ts

```diff
@@ -22,6 +22,7 @@ export interface SMTPServiceConfig {
     };
     port: number;
     secure?: boolean;
+    username?: string;
```

Comment:
Do you think we should rename this variable to `authUsername`? To make it clear that it's for auth and not to be confused with `from.name`?

Reply:
I am in the favour of keeping it as `username` and I see that you have already merged the PR with `username` 👍